### PR TITLE
Crash in our App fix

### PR DESCRIPTION
**Pull request by thegdznet**
java.lang.NullPointerException: Attempt to invoke virtual method
'com.google.android.gms.maps.Projection
com.google.android.gms.maps.GoogleMap.getProjection()' on a null object
reference
at com.airbnb.android.react.maps.AirMapView$13.run(AirMapView.java:643)
at android.os.Handler.handleCallback(Handler.java:739)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5289)
at java.lang.reflect.Method.invoke(Native Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.ja
va:899)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694)

### CODE REVIEW

**Comment by christopherdro** [comment]
lgtm

/cc @felipecsl

File: android/src/main/java/com/airbnb/android/react/maps/AirMapView.java [hunk 1]
```diff
@@ -646,9 +647,12 @@ public void stopMonitoringRegion() {
         @Override
         public void run() {
 
-            LatLngBounds bounds = map.getProjection().getVisibleRegion().latLngBounds;
-            if (lastBoundsEmitted == null ||
-                    LatLngBoundsUtils.BoundsAreDifferent(bounds, lastBoundsEmitted)) {
+            Projection projection = map.getProjection();
+            VisibleRegion region = (projection != null) ? projection.getVisibleRegion() : null;
+            LatLngBounds bounds = (region != null) ? region.latLngBounds : null;
```
Comment (felipecsl) [inline android/src/main/java/com/airbnb/android/react/maps/AirMapView.java]: nit: you don't need the parenthesis around `projection != null` and `region != null`

**Comment by thegdznet** [comment]
Many thanx to accept my pull request. Do You know when the changes will be reflected to master?

Concerning the parenthesis, they are there only for visibility. You can remove them if You prefer.

**Comment by lelandrichardson** [comment]
Thanks for this! Will be cutting a new release wit this in it soon

**Comment by Hoolii** [comment]
I may be wrong but the stacktrace says that the GoogleMap object is null, not the Projection object.
The fix only does a null check on the Projection object, not the GoogleMap object.
I still have a crash because of this.

**Comment by nikolay-radkov** [comment]
I am also having the same problem with the latest release.

**Comment by nikolay-radkov** [comment]
I've created another Pull request for this issue, you can find it here [#1469](https://github.com/airbnb/react-native-maps/pull/1469)

**Pull request merged**